src/modules/nfts/components/wizard/RoyaltiesCreators.tsx

```diff
@@ -0,0 +1,706 @@
+                },
+              ]}
+              // initialValue={royaltiesInput}
+            >
+              <InputNumber<number>
+                min={1}
+                max={100}
+                formatter={(value) => `${value}%`}
+                parser={(value) => parseInt(value?.replace('%', '') ?? '0')}
+                style={{ borderRadius: 4, minWidth: 103 }}
+                onChange={(val: number) => setRoyaltiesInput(val)}
+              />
+            </Form.Item>
+          </Form.Item>
+          {/* Display creators */}
+          {creators.length < MAX_CREATOR_LIMIT && (
+            <Row justify="space-between" align="middle">
+              <Paragraph style={{ fontWeight: 900 }}>Creators split</Paragraph>
+              <StyledClearButton type="text" noStyle onClick={() => toggleCreatorField(true)}>
+                Add Creator
+              </StyledClearButton>
+            </Row>
+          )}
+          <Row>
+            {creators.map((creator) => (
+              <CreatorsRow
+                creatorAddress={creator.address}
+                share={creator.share}
+                isUser={creator.address === userKey}
+                key={creator.address}
+                updateCreator={updateCreator}
+                removeCreator={removeCreator}
+              />
+            ))}
+          </Row>
+          {/* Add creator form */}
+          {showCreatorField && (
+            <Row>
+              <Form.Item
+                name="addCreator"
+                style={{ width: '100%' }}
+                rules={[
+                  { required: true, message: 'Please enter a value' },
+                  { max: 44 },
+                  // need to handle someone pasting in a string longer than 44 char. Right now the input just caps it at 44 without showing anything
+                  { min: 44, message: 'Must be at least 44 characters long' },
+                  {
+                    message: 'All creator hashes must be unique',
+                    async validator(rule, value: string) {
+                      const existingCreators = creators.map((c) => c.address);
+                      const indexOfDuplicate = existingCreators.findIndex((a, i) => value === a);
+                      if (indexOfDuplicate !== -1) {
+                        throw new Error();
+                      }
+                    },
+                  },
+                  {
+                    message: 'Creator address is not valid',
+                    async validator(rule, creatorAddress: string) {
+                      if (creatorAddress.indexOf(' ') >= 0) {
+                        // whitespace detected
+                        throw new Error();
+                      }
+                      return true;
+                      // a bit unsure about this requirement in the end
+                      const base64RegEx =
+                        /^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)?$/;
```

Comment:
creator keys are base58, not base64

Reply:
Do we have a specific set of rules we want to enforce here? right now we just make sure there is no whitespace 😅

Reply:
We also have another rule that says keys need to be minimum 44 char, but my private wallet address only has 43 😱 Is there a known limit?

Reply:
The 44 char limit doesn't come from anywhere, but I do wonder what the requirements for an address are. Does metaplex enforce any?

Reply:
@davetorbeck The pubkey size is derived from the `Ed25519` cipher Solana uses for wallets.  I believe the key is 32 bytes, and then it's base58 encoded, so the character limit comes from that.  I've seen it vary between addresses by one character, so I'm not really sure how the length is derived from the raw binary key...

Reply:
Okay we are gonna mirror metaplex and just allow any string input for now.  Will write follow up test for base58 validation